template/build/core/phing/tasks/deploy.xml

```diff
@@ -170,4 +179,9 @@
       </fileset>
     </delete>
   </target>
+
+  <target name="deploy:acsf:init" description="Re-initialize ACSF with the settings.php changes required for artifact.">
+    <chmod file="${deploy.dir}/docroot/sites/default/settings.php" mode="0755" />
+    <exec dir="${deploy.dir}/docroot" command="${drush.bin} --include=${deploy.dir}/docroot/modules/contrib/acsf/acsf_init acsf-init -y" logoutput="true" checkreturn="true"/>
```

Comment:
It's a paradigm shift from how we originally supported ACSF. Instead of Bolt supporting ACSF and ACE simultaneously out of this box, this adds a flag that you set to add ACSF compatibility "on-the-fly" to a build artifact.

If we're taking this approach, we also need to update the ACSF readme and rip out the ACSF-specific logic in the existing settings.php.